applets/wncklet/window-list.ui

```diff
@@ -386,6 +386,71 @@
                 <property name="position">3</property>
               </packing>
             </child>
+            <child>
+              <object class="GtkBox" id="mouse_scroll_box">
+                <property name="visible">True</property>
+                <property name="can_focus">False</property>
+                <property name="orientation">vertical</property>
+                <property name="spacing">6</property>
+                <child>
+                  <object class="GtkLabel">
+                    <property name="visible">True</property>
+                    <property name="can_focus">False</property>
+                    <property name="label" translatable="yes">Mouse Scroll</property>
```

Comment:
This should be "Mouse Scroll**ing**" instead of "Mouse Scroll"

Reply:
Has been updated